src/test/java/andersen/AndersenTasks/collections/CustomArrayListTests.java

```diff
@@ -0,0 +1,55 @@
+package andersen.AndersenTasks.collections;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+public class CustomArrayListTests {
+    private CustomArrayList customArrayList;
+
+    @BeforeEach
+    public void init(){
+        customArrayList = new CustomArrayList<>();
+    }
+
+    @Test
+    public void testNotNull(){
+        Assertions.assertNotNull(customArrayList);
+    }
+
+    @Test
+    public void testNull(){
+        Assertions.assertEquals(0, customArrayList.getSize());
+    }
+    @Test
+    public void testPut(){
```

Comment:
Same here and for other tests in general, for now it is easy to understand what they are doing by just glance at them, but I strongly recommend to make method names more self descriptive